### PR TITLE
Gentx BwareLabs

### PR DESCRIPTION
**Pull request by silviugae**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by silviugae** [comment]
Hello, @liamsi. Any idea why these 2 checks have been skipped?

Validate Gentx / jq-format (pull_request)
Validate Gentx / valid-gentx (pull_request)

Thank you in advance!

**Review by YazzyYaz — APPROVED** [review]

File: celestia/peers.txt [hunk 1]
```diff
@@ -1 +1 @@
-
+0cb62ebbce54e2d5e72ced753a879d3c08c61583@162.19.171.80:26656
```
Comment (evan-forbes) [inline celestia/peers.txt:1]: please fix conflict by submitting the peer info in a separate file in this directory named as `your-validator.txt`

**Review by evan-forbes — CHANGES_REQUESTED** [review]

File: celestia/peers.txt (hunk 1 above)
Comment (silviugae) [reply celestia/peers.txt]: Made the updates, sorry for the delay, @evan-forbes

**Comment by silviugae** [comment]
All the checks have passed and the peer info was moved to the BwareLabs.txt file.

Please review and approve, @evan-forbes @YazzyYaz @liamsi.

Thank you and have a great day.

**Review by liamsi — APPROVED** [review]

**Review by YazzyYaz — APPROVED** [review]

**Pull request merged**